airbyte-cdk/python/unit_tests/sources/file_based/scenarios/validation_policy_scenarios.py

```diff
@@ -0,0 +1,484 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from airbyte_cdk.sources.file_based.exceptions import ConfigValidationError, FileBasedSourceError
+from unit_tests.sources.file_based.scenarios.scenario_builder import TestScenarioBuilder
+
+_base_single_stream_scenario = (
+    TestScenarioBuilder()
+    .set_files(
+        {
+            "a.csv": {  # The records in this file do not conform to the schema
+                "contents": [
+                    ("col1", "col2"),
+                    ("val_a_11", "val_a_21"),
+                    ("val_a_12", "val_a_22"),
+                ],
+                "last_modified": "2023-06-05T03:54:07.000Z",
+            },
+            "b.csv": {
+                "contents": [
+                    ("col1",),
+                    ("val_b_11",),
+                    ("val_b_12",),
+                ],
+                "last_modified": "2023-06-05T03:54:07.000Z",
+            },
+            "c.csv": {
+                "contents": [
+                    ("col1",),
+                    ("val_c_11",),
+                    ("val_c_12","val_c_22"),  # This record doesn't conform to the schema
+                    ("val_c_13",),
+                ],
+                "last_modified": "2023-06-05T03:54:07.000Z",
+            },
+            "d.csv": {
+                "contents": [
+                    ("col1",),
+                    ("val_d_11",),
+                ],
+                "last_modified": "2023-06-05T03:54:07.000Z",
+            },
+        }
+    )
+    .set_file_type("csv")
+    .set_expected_catalog(
+        {
+            "streams": [
+                {
+                    "json_schema": {
+                        "type": "object",
+                        "properties": {
```

Comment:
shouldn't the schema include the extra fields added by DefaultFileBasedStream (_ab_source_file_last_modified and _ab_source_file_url)?